mmv1/products/securityposture/PostureDeployment.yaml

```diff
@@ -0,0 +1,138 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'PostureDeployment'
+description: |
+  Represents a deployment of a security posture on a resource.
+import_format: ['{{%parent}}/postureDeployments/{{posture_deployment_id}}']
+base_url: '{{parent}}/postureDeployments'
+self_link: '{{parent}}/postureDeployments/{{posture_deployment_id}}'
+
+create_url: '{{parent}}/postureDeployments?postureDeploymentId={{posture_deployment_id}}'
+update_verb: :PATCH
+update_mask: true
+
+
+autogen_async: true
+async: !ruby/object:Api::OpAsync
+  operation: !ruby/object:Api::OpAsync::Operation
+    base_url: '{{op_id}}'
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'securityposture_posture_deployment_basic'
+    primary_resource_id: 'postureDeployment'
+    test_env_vars:
+      org_id: :ORG_ID
+      project_number: :PROJECT_NUMBER
+
+parameters:
+  - !ruby/object:Api::Type::String
+    name: parent
```

Comment:
In our gcloud command which is currently being used by customers we have only parent field, which accepts it in the provided format. Thus have kept it that way.

Reply:
We generally prefer individual url fields, unless it's for consistency with preexisting resources in the product. Since this is a new product, we'd want to split the fields per part. And it wouldn't be necessary to follow the gcloud command format to implement it in Terraform.